include/riak_cs_gc.hrl

```diff
@@ -106,3 +93,14 @@
 -define(DEFAULT_GC_BATCH_SIZE, 1000).
 -define(DEFAULT_GC_WORKERS, 2).
 -define(EPOCH_START, <<"0">>).
+
+-record(gc_manager_state, {
+          next :: undefined | non_neg_integer(),
+          gc_batch_pid :: undefined | pid(),
+          batch_history = [] :: list(#gc_batch_state{}),
```

Comment:
That's nice. What I'm concerned about is indefinite growth of the list.
5min interval (for worse case estimation, somewhat smaller than default) then 8640 entries for a month, 10^5 for a year.